.github/workflows/wheels.yml

```diff
@@ -122,6 +123,65 @@ jobs:
 #          echo $(gcc --version)
 #        if: ${{ runner.os == 'Windows' && env.IS_32_BIT == 'true' }}
 
+      - name: Setup macOS
+        if: matrix.buildplat[0] == 'macos-11' || matrix.buildplat[0] == 'macos-14'
+        run: |
+          if [[ ${{ matrix.buildplat[2] }} == 'arm64' ]]; then
```

Comment:
If the whole step is in this type of conditional, it might make more sense to me to split them into two separate steps. But maybe you're planning on deduplicating and sharing more code across the two variants, in which case +1 for keeping them as a single joint step